server.py

```diff
@@ -1,12 +1,13 @@
-from flask import Flask, jsonify, request
+from flask import Flask, jsonify, request, Response
 from DBConnection2 import DBConnection
 import pymongo
 from bson.json_util import dumps
 from bson import json_util, ObjectId, objectid
+from flask_cors import CORS
 
 app = Flask(__name__)
+CORS(app)
```

Comment:
**Cross-Origin Resource Sharing (CORS)** is a mechanism that uses additional HTTP headers to tell browsers to give a web application running at one origin, access to selected resources from a different origin. A web application executes a cross-origin HTTP request when it requests a resource that has a different origin (domain, protocol, or port) from its own.